### ISSUE TITLE
paper-menu accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-menu#12]
In the column to the right, contrast on "starred" and "inbox" is not sufficient. Is that supposed to be disabled text? 

Windows specific: 
Chrome / NVDA: When using the arrow keys to navigate through, I cannot use the arrows to get from the top set of menus to the bottom multi-select menu. I have to press tab to get me to the bottom menu, and in doing so, I am not able to access the "multi-select" text using the screen reader. This seems to be specific to NVDA on Chrome though.

**Comment by morethanreal** [comment]
The section on menus and menu bars in http://www.w3.org/TR/wai-aria-practices/ says the arrow keys are used to navigate within the menu. It doesn't seem to indicate it should also navigate between menus.

**Comment by lpalmaro** [comment]
Yes, that is true. When navigating with NVDA, using the arrows takes you linearly through the page though (it's different than when using the keyboard without a screen reader). This seemed to work okay on Firefox / NVDA, but for some reason, on Chrome I couldn't get to the "multi-select" text with the screen reader. It worked fine on other platforms with other screen readers though.

**Comment by lpalmaro** [comment]
I suppose this is actually fine though... since it's not the menu I'm having trouble navigating, it's actually just the inability to access the description "multi-select" on the demo page, only with Chrome/NVDA.

**Comment by morethanreal** [comment]
It seems like we can close this?
